### PR TITLE
Char array handling

### PR DESCRIPTION
**Pull request by gaborgsomogyi**
At the moment char array traced like this:
```
TraceAgent (trace_retval): `public static char[] org.apache.hadoop.security.ProviderUtils.locatePassword(java.lang.String,java.lang.String) throws java.io.IOException returns with [C@1fe05fff
```

With the change now it's like this:
```
TraceAgent (trace_retval): `public static char[] org.apache.hadoop.security.ProviderUtils.locatePassword(java.lang.String,java.lang.String) throws java.io.IOException returns with [s, e, c, r, e, t, _, p, a, s, s, w, o, r, d]
```

### CODE REVIEW

File: src/main/java/net/test/interceptor/TraceRetValueInterceptor.java [hunk 1]
```diff
@@ -42,9 +42,15 @@ public Object intercept(
     Object retVal = callable.call();
     long end = (this.logThresholdMs == 0) ? 0 : System.currentTimeMillis();
     if (this.logThresholdMs == 0 || end - start >= this.logThresholdMs) {
+      String retValStr;
+      if (retVal instanceof char[]) {
+        retValStr = new String((char[]) retVal);
```
Comment (attilapiros) [inline src/main/java/net/test/interceptor/TraceRetValueInterceptor.java:47]: Thanks for the PR!

Can you introduce a new action param "stringify_array" false by default (to the  KNOWN_ARGS) and use `Arrays.toString(array)` when it is switched on and do that for all the different types of arrays  (https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#isArray--) ?

Comment (attilapiros) [reply src/main/java/net/test/interceptor/TraceRetValueInterceptor.java:47]: As we discussed offline the param is not needed.

Comment (gaborgsomogyi) [reply src/main/java/net/test/interceptor/TraceRetValueInterceptor.java]: I've added new array to string helper class as we've agreed. Now the result looks like this:
```
TraceAgent (trace_retval): `public static char[] org.apache.hadoop.security.ProviderUtils.locatePassword(java.lang.String,java.lang.String) throws java.io.IOException returns with [s, e, c, r, e, t, _, p, a, s, s, w, o, r, d]
```

**Comment by gaborgsomogyi** [comment]
Adapted the description too.

**Pull request merged**